ql/src/test/results/clientpositive/llap/external_jdbc_table2.q.out

```diff
@@ -713,16 +713,21 @@ STAGE PLANS:
         TableScan
           alias: p
           properties:
-            hive.sql.query SELECT "ikey"
+            hive.sql.query SELECT "ikey", "bkey", "fkey", "dkey"
 FROM "EXTERNAL_JDBC_SIMPLE_DERBY2_TABLE1"
-WHERE ("bkey" = 10 AND "dkey" = 15.15 OR "bkey" = 20 AND "dkey" = 25.25) AND "bkey" IN (10, 20) AND ("dkey" IN (15.15, 25.25) AND "ikey" IS NOT NULL)
```

Comment:
The fact that we are not pushing down the filter to the Jdbc storage handler seems like a regression that we need to investigate further and probably address.

Reply:
This is failing because `SEARCH` operator is not white-listed in `sqlDialect.supportsFunction`, so it fails in `JDBCRexCallValidator.java`. Also, I couldn't find any support for `SEARCH` in JdbcRules. Ideally, this operator should be white-listed and then all `SEARCH` nodes should be expanded (since jdbc engines might not support it) in all the `convert` methods in `Jdbc*Rule` classes in Calcite.

For now, we might have to handle it in Hive.

Reply:
Fixed this in Hive. Please look into JDBCExpandExpressionsRule.java in this PR.